userChomeHideTabs.css

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
FYI: I licensed this repository as GPL, but afaik MPL and GPL are compatible, so this shouldn't be a problem. I have no problem in leaving this header here. (In fact, [GNU recommends this](https://www.gnu.org/licenses/license-list.en.html#MPL-2.0))

Do you want explicit attribution? It was my intent to give it on a per-file basis, but you didn't include it.

Reply:
Update: both GPL and MPL *require* attribution. If you don't add it, I will add your GitHub username instead.